hts-sys/Cargo.toml

```diff
@@ -17,15 +17,16 @@ pre-release-commit-message = "release version {{version}}"
 tag-message = "Version {{version}} of Rust-HTSlib."
 
 [dependencies]
-libc = "0.2"
-libz-sys = "1.0"
+libc = "0.2.68"
+libz-sys = { version = "1.0.25", optional = true }
```

Comment:
I don't think libz-sys should be optional.  It's required for BAM and vcf.gz support and there's no way to turn off those features in hts-lib, so the user shouldn't be allowed to disable that dependency.